tests/runtime/call

```diff
@@ -375,6 +375,11 @@ REQUIRES grep -q '^cgroup2' /proc/mounts
 TIMEOUT 5
 MIN_KERNEL 4.18
 
+NAME strerror
+PROG BEGIN { print(strerror(1)); exit(); }
```

Comment:
Would this work?
```suggestion
RUN {{BPFTRACE}} --include "stderr.h" -e 'BEGIN { print(strerror(EPERM)); exit(); }'
```
Seems more reliable to me.

Reply:
Yes (with `errno.h`).